process/sctfcoil.py

```diff
@@ -5646,6 +5646,20 @@ def outtf(self, peaktfflag):
             )
             po.oblnkl(self.outfile)
 
+        tf_total_height = build_variables.dh_tf_inner_bore + 2 * build_variables.tfcth
+        tf_total_width = (
```

Comment:
Why is the width being calculated by adding the coil thicknesses to the bore height?